test/test_pointwise.cpp

```diff
@@ -430,4 +430,29 @@ TEST_F(PointwiseTest, VIssue1567ectorizationFactorAnalysisCase3) {
   testValidate(fusion, cg_outputs, aten_inputs, __LINE__, __FILE__);
 }
 
+TEST_F(NVFuserTest, FusionTensorRankLimit) {
```

Comment:
```suggestion
TEST_F(PointwiseTest, FusionTensorRankLimit) {
```

Also, `sum` doesn't seem pointwise. Maybe test_gpu_fused_reduction.cpp is a better home?